### DISCUSSION TITLE
"isolcpus=" to enable offline cores Orin NX

### DISCUSSION THREAD

**Discussion by redeexpressos** [OE4T/meta-tegra#1552] [Q&A]
Hi, on Jetson TX2 I was able to enable the offline/denver cores by adding `isolcpus=` to `/boot/extlinux/extlinux.conf`. From what I understand, Orin does not use `grub-efi`, so this solution is not doable.

![image](https://github.com/OE4T/meta-tegra/assets/166127388/257e9f19-cc28-41b5-9c57-bee745022683)

Ive tried the `APPEND += " isolcpus="` yocto option, still no success.

What would be the correct way?

**Comment by kekiefer** [comment]
Refer to the Jetson developer guide for this kind of guidance. You will use nvpmodel to change the online CPU count, among other power-related features. See the section "Supported Modes and Power Efficiency" here: https://docs.nvidia.com/jetson/archives/r35.5.0/DeveloperGuide/SD/PlatformPowerAndPerformance/JetsonOrinNanoSeriesJetsonOrinNxSeriesAndJetsonAgxOrinSeries.html?highlight=nvpmodel#supported-modes-and-power-efficiency
